src/indent.cpp

```diff
@@ -1560,56 +1560,51 @@ void indent_text(void)
          frm.pse[frm.pse_tos].indent_tmp = frm.pse[frm.pse_tos].indent;
          frm.pse[frm.pse_tos].indent_tab = frm.pse[frm.pse_tos].indent;
          log_indent_tmp();
-
          indent_column_set(frm.pse[frm.pse_tos].indent_tmp);
 
-         if ((cpd.settings[UO_indent_class_colon].b && (pc->type == CT_CLASS_COLON)) ||
-             (cpd.settings[UO_indent_constr_colon].b && (pc->type == CT_CONSTR_COLON)))
+         prev = chunk_get_prev(pc);
+         next = chunk_get_next(pc);
```

Comment:
Oh, right, I see, I could simplify that further.
Adds one level of indentation, but makes the logic even easier to follow.
Thanks for the review!